packages/customWidgets/maps-web/src/components/LeafletMap.ts

```diff
@@ -163,7 +163,7 @@ export class LeafletMap extends Component<LeafletMapProps, LeafletMapState> {
 
         return tileLayer(urlTemplate, {
             attribution: mapAttribution,
-            id: mapProvider === "mapBox" ? "mapbox.streets" : undefined
+            id: mapProvider === "mapBox" ? "mapbox/streets-v11" : undefined
```

Comment:
In my original fix there were a few things I also had to do, can you check:

- Change the custom mapbox url: https://github.com/mendix/widgets-resources/pull/533/files#diff-7d06e9f309dae0d16f8d8dab5ac85130287fca52d7bed3010d1190484d95ec06R6
- Add `tileSize: 512` and `zoomOffset: -1` to the map attributes because the previous default was 256: https://github.com/mendix/widgets-resources/pull/533/files#diff-7d06e9f309dae0d16f8d8dab5ac85130287fca52d7bed3010d1190484d95ec06R6

Reply:
Fixed